CoreDataPlatform/Entities/CDAddress+CoreDataProperties.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  CDAddress+CoreDataProperties.swift
+//  CleanArchitectureRxSwift
+//
+//  Created by Andrey Yastrebov on 10.03.17.
+//  Copyright © 2017 sergdort. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+
+extension CDAddress {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<CDAddress> {
+        return NSFetchRequest<CDAddress>(entityName: "CDAddress");
+    }
+
+    @NSManaged public var city: String?
```

Comment:
Ok, I'll check this. These file are xcode autogenerated, and for some reason it always sets optional, regardless the optional checkbox unchecked